eclass-to-manpage.awk

```diff
@@ -22,6 +22,7 @@
 # @SUPPORTED_EAPIS: <optional; space-separated list of EAPIs>
 # @BLURB: <required; short description>
 # @DEPRECATED: <optional; replacement ("none" for no replacement)>
+# @INDIRECT_ECLASSES: <optional; space-separated list of allowed indirect eclasses>
```

Comment:
I would have named them 'transitive inherits' or 'reexported APIs' but w/e. I think any name will be confusing.

Reply:
I came to the same conclusion and also don't care what it's called, perhaps mailing list responses will have suggestions.